MXLPersonalArmoryHook/include/PipeClient.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <Windows.h>
+#include <string>
+
+#define DEFAULT_PIPE_ADDRESSW L"\\\\.\\pipe\\MXLPersonalArmoryHook"
+#define DEFAULT_PIPE_ADDRESS "\\\\.\\pipe\\MXLPersonalArmoryHook"
```

Comment:
Is `DEFAULT_PIPE_ADDRESS ` used anywhere?

Reply:
Not at the moment, but might be needed in the future. This way is consistent with Microsoft's ANSI/Wide versions of everything involving strings.